src/data/matrix/basic.lean

```diff
@@ -878,6 +882,13 @@ by { ext, refl }
 
 end transpose
 
+section has_star
+variables [has_star α]
+instance : has_star (matrix n n α) := {star := conj_transpose}
+lemma conj_transpose_eq_star_of_square_matrix (M : matrix m m α) :
+star M = Mᴴ  := rfl
```

Comment:
For the [fourth time](https://github.com/leanprover-community/mathlib/pull/8291#discussion_r668717838)

```suggestion
lemma star_eq_conj_transpose (M : matrix m m α) : star M = Mᴴ := rfl
```